include/specfem_mpi/specfem_mpi.hpp

```diff
@@ -18,7 +18,10 @@ namespace MPI {
  * Incase specfem is compiled without MPI then I need placeholders for reducer
  * types
  */
-enum reduce_type { sum = MPI_SUM, min = MPI_MIN, max = MPI_MAX };
+using reduce_type = MPI_Op;
+const MPI_Op sum = MPI_SUM;
```

Comment:
I think you'd need `constexpr static MPI_Op` here.

Reply:
I think the compilation will fail when MPI is enabled if tis a `const MPI_OP` itd have it be a static variable. Constexpr since we already know it at compile time

Reply:
![image](https://github.com/user-attachments/assets/5a669c83-11c3-41df-bfce-aca3f5c0f0c7)
constexpr wouldn't compile